examples/morpheusvm/tests/workload/genesis.go

```diff
@@ -45,10 +53,13 @@ func NewGenesis(keys []ed25519.PrivateKey, minBlockGap time.Duration) *genesis.D
 	genesis.Rules.MaxBlockUnits = fees.Dimensions{1800000, math.MaxUint64, math.MaxUint64, math.MaxUint64, math.MaxUint64}
 	genesis.Rules.MinBlockGap = minBlockGap.Milliseconds()
 
+	genesis.Rules.NetworkID = uint32(1)
+	genesis.Rules.ChainID = ids.GenerateTestID()
```

Comment:
What's the reason for these changes?

Reply:
When we call `GenerateTransaction`, we need to know the `ChainID`, which is included in the `Transaction.Base`.
For `integration` testing, the code in `tests/integration/integration.go` was generating this random ChainID and using it. Instead, I moved it one step up into the testVM.
Doing so allowed me to avoid the unnecessary `parser, err := lcli.Parser(ctx)` call by using the already-known ( or should've-been-known ) values.